automation/common.sh

```diff
@@ -52,9 +52,9 @@ runapi () {
 		-e DEVICE_CONFIG_OPENVPN_CA='LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNXRENDQWNHZ0F3SUJBZ0lKQVBTeHZhSG5zanVpTUEwR0NTcUdTSWIzRFFFQkJRVUFNRVV4Q3pBSkJnTlYKQkFZVEFrRlZNUk13RVFZRFZRUUlEQXBUYjIxbExWTjBZWFJsTVNFd0h3WURWUVFLREJoSmJuUmxjbTVsZENCWAphV1JuYVhSeklGQjBlU0JNZEdRd0hoY05NVE14TWpFeU1UUTBOelUyV2hjTk1qTXhNakV3TVRRME56VTJXakJGCk1Rc3dDUVlEVlFRR0V3SkJWVEVUTUJFR0ExVUVDQXdLVTI5dFpTMVRkR0YwWlRFaE1COEdBMVVFQ2d3WVNXNTAKWlhKdVpYUWdWMmxrWjJsMGN5QlFkSGtnVEhSa01JR2ZNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0R05BRENCaVFLQgpnUURsTXZRMmp1WnJ6WFJxV3BYN3Q0RlhYTGw0RzhuY05UMXYyTW1UM3BwNnVGNG5rVkd1UjRZdFczYmlwQ0thClRYRnZ5aFp1eEUvN2ZKWUdoYWZNV1pzMjZrUHQ3dnNtaVRSRUVHQytCSHFOUWIwd0ltckxaT0syVzk3R2R1U2UKZThuWmNXU0MzWjhVQ1hSQkg3WmtzNHphRndodGNnZ3ZkSi9Qdzl3MTJ0Tkl6UUlEQVFBQm8xQXdUakFkQmdOVgpIUTRFRmdRVVU0V3FYMmZMeDdnVTJRcFF2VkgwblpOUXNSWXdId1lEVlIwakJCZ3dGb0FVVTRXcVgyZkx4N2dVCjJRcFF2VkgwblpOUXNSWXdEQVlEVlIwVEJBVXdBd0VCL3pBTkJna3Foa2lHOXcwQkFRVUZBQU9CZ1FBWWhUTWQKUHNDQ3hIbHFCak91c3dQNlBZT2c1TXo1QXFnNzBaWmZoTGpFV3lvU0VzclZKTTRlcyt4cnRIQUl0VDI4QXhreQpSUE43cnpMc2QzR2lIOWE2V3phZU5kcFdkWU1TaTMrTnJOYmtPU3ZuTmhHeHUvUUhiMEx0bWV0eHBENlNERmZQCkoxMUVuTjM0dldHMUpCWUh2NVNvditFOTkzclJKdkU0VUV1bFpRPT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=' \
 		-e DEVICE_CONFIG_SSH_AUTHORIZED_KEYS='ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCc5cLVsu7b52tzxwa8A6cs4LC2I+IMtyKeVZFVAWdhfEWGEJL78rn0tC4YmHBSWW0hBeZmn8f7ZKcZ/r1W2RpuH7wpUc56/3hffJ0GoktMOX69pjNsE3rWr8/cZbl/LaJjeHBpyFZ6VimYl5KiYyHJH7RZwlEYYiJ2YDCbte51jvmotFTqiOJNPCM3SXWFjsUGDOWwNi7GDxG+ulb0cBnYzyYlHTj5dS/uya0v8S267TYO2jZ9lUvZHYYQBilYoN3qT79B/lRJq/AKsaSPI2VWkwoAWLObJRjQjoIsClxcfaSfRg6fgdezv3saeorVMIdEYAj3CQVmLszCNGC8dgqB9OoKdjd/OUG/bFUc86S+y0dZX3vqKBfZjE1dHNzNkQeulsbRj0FGFTxFYL0Ng7jlr98iXusnNg236LGsBTeod2MLfMRsrOc0Utfc5Pq2Zkyak4aJ6URBFqZnKlJwvI3AFznxvHD1PmIrwzCCmLA409v3WbXEK4KZrUIs81wlQWAK3919ViozdnR23f4hZtq3P5M6AbnMMdIOB1jDjPK//eaTMf3erHVa0nTMMwScBDnrT86yJBBEwpUVpIp6YUBFaVMQscX3gBTd+RyOU9jOvyinGjosJfF38qsMJTNHh48zgic5sSLhtPQgkUjWhpcnfjqrLRewKbVd+VZQc1WsdQ==' \
 		-e IMAGE_MAKER_URL=https://img.balenadev.io \
-		-e IMAGE_STORAGE_BUCKET=balena-dev-img \
-		-e IMAGE_STORAGE_ENDPOINT=s3.balenadev.io \
-		-e IMAGE_STORAGE_PREFIX=images \
+		-e IMAGE_STORAGE_BUCKET=resin-production-img-cloudformation \
+		-e IMAGE_STORAGE_PREFIX=resinos \
+		-e IMAGE_STORAGE_ENDPOINT=s3.amazonaws.com \
```

Comment:
Do tests depend on making actual calls to S3?

Reply:
Yes. If you make use of the `applications` fixture then it will make an S3 call to determine device-types. Without mocking the S3 calls, this is needed.

Reply:
I am going to mock the S3 calls 👍